protocol/src/main/java/org/apache/drill/exec/proto/beans/CoreOperatorType.java

```diff
@@ -58,7 +58,8 @@
     HBASE_SUB_SCAN(33),
     WINDOW(34),
     NESTED_LOOP_JOIN(35),
-    AVRO_SUB_SCAN(36);
+    AVRO_SUB_SCAN(36),
```

Comment:
I think the queries may run fine, but the use of this protobuf definition if I'm not mistaken is to generate query profiles. If you don't return a value that Drill knows about from the method  getReaderOperatorType(), then the operator type will show up as unknown in the profile. I should have been clearer, my comment was more a question to @StevenMPhillips and @jacques-n. I think we may need to refactor this interface to just return a String and not use the protobuf enum, or have a separate "notes" field if we want to have a generic filescan operator ID in this enum with extra data to say what kind of scan we used.